### ISSUE TITLE
Import-NAVApplicationObject failure

### ISSUE THREAD

**Issue by fvet** [NAVBaaS/GitIntegration#7]
Scenario:

- I've created a new Cronus container
- Exported all our licensed ISV objects (+ other ISV objects) to the Modified folder
- Ran 'Sync Preview' > No Errors (since no real import of objects is done)
- Ran 'Sync.' > Error (due to my mistake)

```
VERBOSE: Creating text file to import.
VERBOSE: Importing objects.
Import-NAVApplicationObject : : [18023700] You do not have permission to create the 'EM Status Report' Report.
...
```

This concerned a Continia Document Expense object for which we have no 'Insert' rights in our license. To overcome this issue, I had to import the Continia objects as a fob file first (which makes perfectly sense). However, since no other objects had been imported, I reran the 'Sync.' command, but no changes were detetected.

```
Started function: Synchronize container.
VERBOSE: Repository commit hash is 54daf4f9f1b252166c0989f172c07c8cedaa379b
VERBOSE: Container commit hash is 54daf4f9f1b252166c0989f172c07c8cedaa379b
VERBOSE: Nothing to import, nothing to export and nothing to delete.
Finished function: Synchronize container.
```

Importing a fob file with all my modified objects fixed the issue as well.

I'm wondering if I (or colleagues) encounter similar (import txt file) issues in the future - and not really analysing the output windows in VS Code - how I can ensure all changes have actually been imported in my container? Wouldn't it be better to let the sync. container fail in case an import of txt files fails? And let me fix the problems with fob files first. Would that not result in an updated container commit hash?

**Comment by richardrobberse** [comment]
The commit hash shouldn't be updated in case the import fails, I'll have a look as soon as I'm back from vacation (beginning of next week) and hopefully fix it right away.

**Comment by richardrobberse** [comment]
Which Docker image are you using?
I know that in older versions (NAV 2016) errors sometimes look like errors but they're not ending up in powershell's error stream.

Btw: you can always remove or set a different commit hash in your user settings, this is not something I advice you to do but it can be handy to avoid or reproduce things.

**Comment by fvet** [comment]
The docker image used was nav2018-w1-cu6

**Comment by richardrobberse** [comment]
I've found a solution and I will implement it soon.
Thanks for reporting it :)

**Comment by richardrobberse** [comment]
Fixed in version 1.6.0.
